### PR TITLE
Create new feed-variant 20220422

### PR DESCRIPTION
**Pull request by maestromac**
## What type of PR is this? (check all applicable)
- [x] Feature

## Description
Add a new feed-variant as per Jennie's request. Shoutout to @jeremyf and @msarit 

## Related Tickets & Documents
n/a

## QA Instructions, Screenshots, Recordings
n/a

## Added tests?
- [x] no, because they aren't needed

## Added to documentation?
- [x] no documentation needed

## [optional] Are there any post deployment tasks we need to perform?
n/a

### CODE REVIEW

File: config/feed-variants/20220422-jennie-variant.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    "matching_positive_tags_intersection_count": {
+      "cases": [
+        [0, 0.7],
+        [1, 0.85],
+        [2, 0.93],
+        [3, 0.98],
+        [4, 1]
+      ],
+      "fallback": 1
+    },
```
Comment (maestromac) [inline config/feed-variants/20220422-jennie-variant.json:89]: New change here

**Review by msarit — APPROVED** [review]

File: app/models/articles/feeds.rb [hunk 2]
```diff
@@ -189,7 +189,7 @@ def self.lever_catalog
                                             AND comments.deleted = false
                                             AND comments.created_at > :oldest_published_at"])
 
-      relevancy_lever(:matching_negative_tag_intersection_count,
+      relevancy_lever(:matching_negative_tags_intersection_count,
```
Comment (jeremyf) [inline app/models/articles/feeds.rb:192]: Good catch on consistent naming!

**Review by jeremyf — CHANGES_REQUESTED** [review]
It may look like I want to request changes, but really I don't want to accidentally merge this.  @JennieOcken wants this to go live on Monday.

**Comment by jeremyf** [comment]
For those reading along, we don't need tests because in [spec/services/articles/feeds/variant_query_spec.rb](https://github.com/forem/forem/blob/main/spec/services/articles/feeds/variant_query_spec.rb) we test all variants in that directory.

**Review by jeremyf — APPROVED** [review]

**Pull request merged**